### PR TITLE
Fix/67430 wpml event slug support

### PR DESCRIPTION
**Pull request by lucatume**
Ticket: https://central.tri.be/issues/67430

This PR adds the base needed to support WPML slug translation in TEC and PRO; PRO PR here https://github.com/moderntribe/events-pro/pull/402

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task | [Main.php:3048](https://github.com/moderntribe/the-events-calendar/blob/c7a948a9272fdf267d2d089033737dd41b42d89f/src/Tribe/Main.php#L3048)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | There must be a single space after the first parenthesis | [Utils.php:22](https://github.com/moderntribe/the-events-calendar/blob/0b8711e59bbeb1422ff0f648b1dd61ef9c602974/src/Tribe/Integrations/WPML/Utils.php#L22)
:x: | There must be a single space before the last parenthesis | [Utils.php:22](https://github.com/moderntribe/the-events-calendar/blob/0b8711e59bbeb1422ff0f648b1dd61ef9c602974/src/Tribe/Integrations/WPML/Utils.php#L22)
:x: | There must be a single space after the first parenthesis | [Utils.php:24](https://github.com/moderntribe/the-events-calendar/blob/0b8711e59bbeb1422ff0f648b1dd61ef9c602974/src/Tribe/Integrations/WPML/Utils.php#L24)
:x: | There must be a single space before the last parenthesis | [Utils.php:24](https://github.com/moderntribe/the-events-calendar/blob/0b8711e59bbeb1422ff0f648b1dd61ef9c602974/src/Tribe/Integrations/WPML/Utils.php#L24)
:x: | There must be a single space before the last parenthesis | [Utils.php:28](https://github.com/moderntribe/the-events-calendar/blob/0b8711e59bbeb1422ff0f648b1dd61ef9c602974/src/Tribe/Integrations/WPML/Utils.php#L28)
:x: | No space after opening parenthesis is prohibited | [Utils.php:28](https://github.com/moderntribe/the-events-calendar/blob/0b8711e59bbeb1422ff0f648b1dd61ef9c602974/src/Tribe/Integrations/WPML/Utils.php#L28)
:x: | There must be a single space after the first parenthesis | [Utils.php:28](https://github.com/moderntribe/the-events-calendar/blob/0b8711e59bbeb1422ff0f648b1dd61ef9c602974/src/Tribe/Integrations/WPML/Utils.php#L28)
:x: | There must be a single space after the first parenthesis | [Utils.php:28](https://github.com/moderntribe/the-events-calendar/blob/0b8711e59bbeb1422ff0f648b1dd61ef9c602974/src/Tribe/Integrations/WPML/Utils.php#L28)
:x: | There must be a single space before the last parenthesis | [Utils.php:28](https://github.com/moderntribe/the-events-calendar/blob/0b8711e59bbeb1422ff0f648b1dd61ef9c602974/src/Tribe/Integrations/WPML/Utils.php#L28)
:x: | ! must be followed by a single space | [Utils.php:28](https://github.com/moderntribe/the-events-calendar/blob/0b8711e59bbeb1422ff0f648b1dd61ef9c602974/src/Tribe/Integrations/WPML/Utils.php#L28)
:x: | There must be a single space before the last parenthesis | [Utils.php:28](https://github.com/moderntribe/the-events-calendar/blob/0b8711e59bbeb1422ff0f648b1dd61ef9c602974/src/Tribe/Integrations/WPML/Utils.php#L28)
:x: | No space before closing parenthesis is prohibited | [Utils.php:28](https://github.com/moderntribe/the-events-calendar/blob/0b8711e59bbeb1422ff0f648b1dd61ef9c602974/src/Tribe/Integrations/WPML/Utils.php#L28)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Integrations/WPML/Utils.php [hunk 1]
```diff
@@ -11,11 +11,24 @@ class Tribe__Events__Integrations__WPML__Utils {
...
 	 */
-	public static function get_wpml_i18n_strings( array $strings ) {
+	public static function get_wpml_i18n_strings( array $strings, $locale = null, array $domains = null ) {
+		array_multisort($strings);
+		$cache = new Tribe__Cache();
+		$cache_key = 'wpml-i18n-strings_' . serialize($strings);
+
+		$cached_translations = $cache->get_transient( $cache_key, 'wpml_updates' );
+
+		if (!empty( $cached_translations)) {
```
Comment (borkweb) [inline src/Tribe/Integrations/WPML/Utils.php]: There's spacing issues on this line, 24, and 22

**Comment by borkweb** [comment]
Other than spacing (and the merge conflict), this looks pretty good. :+1:

**Comment by lucatume** [comment]
Found another issue, fixing...

**Pull request merged**